### ISSUE TITLE
Make Chip component 'checked' prop controlled

### ISSUE THREAD

**Issue by BeeMargarida** [svelteuidev/svelteui#402]

**Comment by khalibloo** [comment]
Looks like this is already done?
![image](https://github.com/svelteuidev/svelteui/assets/6402969/d25db59b-010a-4466-b2c3-2040a1c5b96b)

**Comment by BeeMargarida** [comment]
It's not, this happens when we change the state inside (like toggling the checked) and then we try to change outside. There's an example of the problematic behaviour here: https://discord.com/channels/954790377754337280/1109187255903453245/1109442589138100335

**Comment by bmw2621** [comment]
I dont believe this is an issue with the component, as the same issue applied to a checkbox yields the same result.  We need to research `createEventForwarder` and see why the `checked` closure isn't passing through.

**Comment by BeeMargarida** [comment]
Yes, it yields the same issue, but we want it to be possible to override the "state" when changing the prop, even if we changed it internally.
